### PR TITLE
devtools: Use ronn-nodejs package on CentOS

### PR DESCRIPTION
**Pull request by mamedin**
Connects to https://github.com/archivematica/Issues/issues/979

### CODE REVIEW

File: tasks/am-devtools.yml [hunk 1]
```diff
@@ -31,21 +31,12 @@
...
-      with_items:
-        - { name: "psych", version: "2.2.4" }
-        - { name: "ronn", version: "" }
+        - "nodejs-ronn"
 
     # Needed to find the command ronn when running make
     - name: "Create ronn symlink (RH/CentOS)"
       file:
-        src: "/usr/local/bin/ronn"
+        src: "/usr//bin/ronn-nodejs"
```
Comment (luisilara) [inline tasks/am-devtools.yml:39]: A cosmetic change, the path has a double slash

**Review by luisilara — APPROVED** [review]
LGTM!

File: tasks/am-devtools.yml (hunk 1 above)
Comment (luisilara) [reply tasks/am-devtools.yml:39]: Tested on CentOS 7 and worked as spected

**Comment by mamedin** [comment]
Cherry-picked to stable/1.11.x: https://github.com/artefactual-labs/ansible-archivematica-src/commit/ea80e1650ceb7fb242cb47d2b20a844a064856f2

**Pull request merged**